Directory.Build.targets

```diff
@@ -36,8 +37,10 @@
         <PackageReference Update="Microsoft.Reactive.Testing" Version="4.3.2"/>
         <PackageReference Update="MediatR" Version="8.0.1"/>
         <PackageReference Update="MediatR.Extensions.Microsoft.DependencyInjection" Version="8.0.0"/>
+        <PackageReference Update="System.IO.Pipelines" Version="4.7.1"/>
+        <PackageReference Update="Nerdbank.Streams" Version="2.4.60"/>
```

Comment:
We actually depend on this in VS to enable our LSP experience there. /cc @ToddGrun

Reply:
I just found them and it was code I didn't have to 100% comprehend (I'm slowly catching on...)